pages/research/mcdr-tools-about.mdx

```diff
@@ -0,0 +1,176 @@
+    ],
+    [
+      'Air-sea CO₂ flux',
+      'mol/m²/yr',
+      'Both',
+      'The movement of carbon dioxide between the atmosphere and the ocean. Negative values indicate ocean CO₂ uptake.',
+    ],
+    [
+      'pH',
+      'n/a',
+      'Both',
+      'The measurement of acidity, or free hydrogen ions, in surface waters. The lower the pH value, the more acidic the seawater.',
+    ],
+  ]}
+
+/>
+
+<Box id='comparing-oae-and-dor' sx={{ scrollMarginTop: '30px' }} />
+
+## Comparing OAE and DOR
+
+We initially defined OAE efficiency (𝜂) as the amount of CO₂ removed from the atmosphere per unit of alkalinity added. However, to compare across DOR and OAE, we use a different efficiency metric (γ): the amount of CO₂ absorbed from the atmosphere per unit of removal potential generated by the intervention.
+
+For DOR, the maximum potential removal is equivalent to the amount of CO₂ extracted from the surface ocean. For OAE, this comparison requires calculating the maximum removal potential associated with the simulated alkalinity additions. This is done as follows. Our original definition of efficiency, the amount of CO₂ removed from the atmosphere per unit of alkalinity added, can be written as
+
+> 𝜂(t) = ∆DIC(t) / ∆Alk
+
+where ∆Alk is the amount of alkalinity added to the ocean and ∆DIC(t) is the resulting
+change in the ocean carbon inventory over time. Under the condition of full re-equilibration,
+we have
+
+> 𝜂max = ∆DIC(t=∞) / ∆Alk
+
+where ∆DIC(t=∞) is the amount of potential DIC uptake induced by the chemical effects of the alkalinity addition. Therefore, to compute an efficiency metric for OAE that is directly comparable to DOR, we divide 𝜂(t) by 𝜂max, yielding
+
+> γOAE(t) = 𝜂(t) / 𝜂max
+
+To perform this conversion on the OAE dataset, we estimate 𝜂ₘₐₓ from the background ocean state, using the thermodynamic equations defining the seawater carbonate system equilibrium chemistry. Notably, 𝜂ₘₐₓ varies from about 0.7 in the tropical surface ocean to about 0.9 at high latitudes. This approach is not perfect, as we are using time-averaged fields and mixing along the trajectory of the alkalinity plume entrains waters with distinct chemical composition. In practice, however, we find that the errors introduced by these factors are very small (less than ±0.1) relative to the large-scale spatial and seasonal structure in the efficiency fields, so the approach provides a robust basis to compare the efficiency of OAE and DOR.
+
+## Version history
+
+<Table
+  columns={[5]}
+  start={[[1], [2], [4]]}
+  width={[[1], [2], [2]]}
+  index={false}
+  sx={{
+    'tr:first-child': {
+      td: {
+        color: 'blue',
+        fontFamily: 'heading',
+        textTransform: 'uppercase',
+      },
+    },
+    'td:first-of-type': { color: 'blue', fontFamily: 'mono' },
+  }}
+  data={[
+    ['Date', 'OAE', 'DOR'],
+    ['10-15-2024', 'Initial release', 'n/a'],
```

Comment:
Super small [consistency](https://github.com/search?q=org%3Acarbonplan+n%2Fa&type=code) thing: can we uppercase to `N/A` in both places on this page?